src/serde.rs

```diff
@@ -55,15 +57,19 @@ impl<K, V> Visitor for LinkedHashMapVisitor<K, V>
 {
     type Value = LinkedHashMap<K, V>;
 
+    fn expecting(&self, formatter: &mut Formatter) -> FmtResult {
+        write!(formatter, "linked hash map")
```

Comment:
I would prefer to keep it simple and say `"a map"` here. This is what we do for the built-in impls - BTreeMap and HashMap.

An example of a message you get from this is:

> invalid type: string "xyz", expected linked hash map

The user is not responsible for figuring out how to represent a linked hash map in their format. They should just see:

> invalid type: string "xyz", expected a map